### ISSUE TITLE
Please clarify the purpose of the backing plate

### ISSUE THREAD

**Issue by DenysGonchar** [GliaX/tourniquet#16]
There is an urgent need for tourniquets in Ukraine right now. I'm trying to organize a small non-commercial manufacturing according to the GliaX/tourniquet design, there are other volunteers interested in this as well. The design looks simple, however I have a couple of questions:

- we did one sample for testing, and it seems that backing plate is the most fragile element (testing was done by professional paramedic).

- backing plate got damaged during the first tightening, and it got completely broken after a couple of attempts.

- it seems that the main problem is that backing plate bends too much during tightening (we tried it on average size men's arm).

- if the backing plate is broken, it blocks (a bit) rotation of the windlass rod. This might be critical in emergency situations.

- the weakness of the backing plate makes reusage of the device impossible. I believe it's supposed to be a one time usage device, but if you don't have access to anything else, then reusage might be an option.

- a general  impression that tourniquet works well even w/o the backing plate. 

- the only obvious purpose for it that we (me and the paramedic) can see is keeping the windlass rod close to the clip. Do we understand the design correctly?

- if it's so, can we just get rid of the backing plate and limit the movement  of the windlass rod somehow differently, e.g. by adding hook/loop tape between the buckle and rod?

Also, according to this photo, there was no backing plate in initial design:
![image](https://user-images.githubusercontent.com/23631268/159893972-6a6f8941-65d8-4a1d-8042-1af89d34b472.png)

Thanks in advance.
Denys

**Comment by DenysGonchar** [comment]
Also, I can make a Russian translation of assembly instructions if you don't have it yet (the current status is `pending`).

Please let me know if you need it, and how it can be submitted.

**Comment by tareko** [comment]
Hello @DenysGonchar . Please accept my apologies for missing this issue. While I've been active on the github, I didn't notice issues - mistake fixed with notifications now.

1. Russian translation - YES PLEASE! If you want, submit it as a pull request and I'll add it

**Comment by tareko** [comment]
>* we did one sample for testing, and it seems that backing plate is the most fragile element (testing was done by professional paramedic).
>* backing plate got damaged during the first tightening, and it got completely broken after a couple of attempts.

This indicates either a materials or an assembly error. If you post photos of the problem, I'll take a quick look and give feedback.

>* it seems that the main problem is that backing plate bends too much during tightening (we tried it on average size men's arm).

The bending is intended. If it does not bend, then it creates a dead space under which blood could flow and puts too much pressure on the area directly under the plastic 'bar' rather than distributing it along the entire length of the backing support.

Also, breakage as above is not a _functional problem_. To answer the topic, *the purpose of the backing plate is to make the experience more comfortable*. It does this by reducing skin pinching and catching under the twisting of the tourniquet. In the case of a nearly lost limb, it doesn't matter, but it's still a 'nice to have' that all modern tourniquets try to do.

> *if the backing plate is broken, it blocks (a bit) rotation of the windlass rod. This might be critical in emergency situations.

This breakage seems to be in a way that I'm not visualizing well. Could you clarify with a photo or more description?

> the weakness of the backing plate makes reusage of the device impossible. I believe it's supposed to be a one time usage device, but if you don't have access to anything else, then reusage might be an option.

We reused these devices in Gaza when we were desperate and internally let them go up to 10 times, but there are lots of reasons why this should _not_ happen. Infection risk through blood-borne illness is one, and the fact tourniquets aren't designed for multiple uses is another.

>* a general impression that tourniquet works well even w/o the backing plate.

This is a correct impression. Just consider the comfort aspect.

>* the only obvious purpose for it that we (me and the paramedic) can see is keeping the windlass rod close to the clip. Do we understand the design correctly?

Not quite. I will say more below on the 'soft' backing plate. The two main reasons in order of importance are: 1) To prevent catching of the skin under the windlass; 2) To keep the orientation and prevent twisting of the bands as they're being drawn tighter.

> If it's so, can we just get rid of the backing plate and limit the movement of the windlass rod somehow differently, e.g. by adding hook/loop tape between the buckle and rod?

The photo you cited above was one of the major field trials where we put tourniquets out with a hard, non-plastic backing in Gaza. During that time, it was done due to a lack of printing time and plastic in the context of needing hundreds of units per week for the Great March of Return, a set of weekly protests where ultimately there were over 22k casualties and over 5k gunshot wounds. It worked OK, but not amazing, which is why we abandoned the idea.

In short, let's review why your backing plates are breaking. It's almost certainly related to an issue with assembly or printing. If you take detailed photos, happy to take a look.

tarek : )

**Comment by tareko** [comment]
Closed by accident

**Comment by jenwilsonglia** [comment]
Will be closing this issue once we upload the Backplate Breakage Report next week regarding this.
